examples/ansibleee-extravolumes.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: redhat.com/v1alpha1
+kind: AnsibleEE
+metadata:
+  name: ansibleee-test
+  namespace: openstack
+spec:
+  # We can specify either playbook, which will run with default ansible-runner options,
+  # or args, which allows specify the whole command that we want to execute
+  playbook: "test.yaml"
+  # args: ["ansible-runner", "run", "/runner", "-vvv", "-p", "test.yaml"]
+  image: "quay.io/jlarriba/openstack-tripleo-ansible-ee"
```

Comment:
`quay.io/jlarriba/openstack-tripleo-ansible-ee`

Reply:
I tested this w/ https://github.com/openstack-k8s-operators/lib-common/pull/88 as described in https://gist.github.com/fmount/ffc4cd6a048cafe2a38ae5f8727e31f9 and it worked